src/sdgio-edit.owl

```diff
@@ -2727,6 +2749,22 @@ AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020235> "Mult
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020235> "improvement of natural resources and ecosystem services system level outcome"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020232> <http://purl.unep.org/sdg/SDGIO_00020235>)
 
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020238> (Increased resilience of the poor to climate change and other shocks)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020238> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020238> "2019-04-28T12:16:49Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.unep.org/sdg/SDGIO_00020238> "1"^^xsd:integer)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020238> "Increased resilience of the poor to climate change and other shocks"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020238>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020239> (Enhanced smallholder market access)
```

Comment:
Replace capital letters with lowercase

Reply:
Done. Thanks for noticing all these important things.